### PR TITLE
Add comments heading

### LINKED ISSUE #6338: Add comment headings to pages with comments
Partial fulfillment for #6200:

- [x] add "comments" heading to PNI product pages
- [ ] add "comments" heading to Blog pages, only visible for blog pages that have comments enabled through the CMS.

### PR DESCRIPTION
**Pull request by Pomax**
Closes #6338

Related PRs/issues #6200

If `USE_COMMENTO` is enabled, this change adds the "Comments" heading to PNI product pages, as well as Blog pages (but only for blog pages that have commenting enabled in the CMS)

Note that the demo URLs don't show an actuall comment box because Commento is a paid service that needs to be bought per-domain (and each review app has its own domain... )

### PNI example

https://foundation-s-add-commen-iheucc.herokuapp.com/en/privacynotincluded/follow-art-response-answer-interest-catch-hit/

image

### Blog example

only kicks in when the comments are enabled in the CMS for a blog post. Example: https://foundation-s-add-commen-iheucc.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/

image

### CODE REVIEW

**Review by mmmavis — APPROVED** [review]
Code looks good. I will defer design review to @sabrinang .

@Pomax can you include links to test pages or screencaps for Sabs to 👀 ?

**Comment by sabrinang** [comment]
Yes please leave links as I'm trying to check out:
https://foundation-s-add-commen-iheucc.herokuapp.com/en/privacynotincluded/
https://foundation-s-add-commen-iheucc.herokuapp.com/en/blog/
but I'm getting this so I don't know if I'm looking at the right spot
image

**Review by TheoChevalier — CHANGES_REQUESTED** [review]
Can we localize these strings please, even though the comment tool is not (yet) localized?

**Comment by Pomax** [comment]
@sabrinang  @TheoChevalier I've updated the app settings, so it should work now. Links and screens in the top comment.

**Review by TheoChevalier — APPROVED** [review]
Thanks!

**Review by sabrinang — COMMENTED** [review]
- can the comment heading/section come after "Related products"?
image

- for the blog context, the heading should match the style of "Relate content":
image

**Comment by Pomax** [comment]
@sabrinang it can, does it need to be full width, then? Because the related products have a different width from the content above them.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html [hunk 1]
```diff
@@ -36,6 +36,7 @@ <h1 class="h1-heading">{{ page.title }}</h1>
     </div>
 
     {% if show_comments %}
+      <h2 class="h2-heading">{% trans "Comments" %}</h2>
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html:39]:
```suggestion
      <h2 class="h4-heading">{% trans "Comments" %}</h2>
```

**Comment by Pomax** [comment]
@sabrinang https://foundation-s-add-commen-vndh71.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/ is with the blog heading set to h4-heading, just like related:

screenshot 725

**Review by sabrinang — APPROVED** [review]
Thanks for making those headings match Pomax!

**Pull request merged**